baseapp-chats/baseapp_chats/graphql/object_types.py

```diff
@@ -98,6 +98,19 @@ class BaseChatRoomObjectType:
     image = ThumbnailField(required=False)
 
     def resolve_all_messages(self, info, **kwargs):
+        if self.participants_count > 2:
```

Comment:
since we are introducing `is_group` in [another PR](https://github.com/silverlogic/baseapp-backend/pull/198/files) we should base on that instead of `participants_count`